src/scrabbleScore.js

```diff
@@ -24,14 +24,33 @@ let SCRABBLE_TILE_SCORES = {
  */
 
 function scrabbleScore(word) {
-  // This is your job. :)
+  let workingWord = word.toLowerCase();
+  let separateLetters = workingWord.split("");
+  let score = 0
+  let counter = 0
+  let key = Object.keys(SCRABBLE_TILE_SCORES);
+  let value = Object.values(SCRABBLE_TILE_SCORES);
+
+  while (counter != workingWord.length){
```

Comment:
You can iterate over the letters in a string directly. `SCRABBLE_TILE_SCORES['k']` gives you the score of the letter `k`, for example.

```javascript
for (let char of word) {
  score += SCRABBLE_TILE_SCORES[char];
}
```